packs/data/pathfinder-society-boons.db/wayfinder-radiant.json

```diff
@@ -0,0 +1,39 @@
+{
+    "_id": "J6lNh2fUQ6jYTRak",
+    "img": "systems/pf2e/icons/default-icons/mystery-man.webp",
+    "name": "Wayfinder, Radiant",
+    "system": {
+        "actionType": {
+            "value": "action"
+        },
+        "actions": {
+            "value": 2
+        },
+        "category": "pfsboon",
+        "description": {
+            "value": "<p><strong>[Item]</strong></p>\n<p><strong>Prerequisites</strong> Liked by Radiant Oath; You own a <em>@UUID[Compendium.pf2e.equipment-srd.Wayfinder]{Wayfinder}</em></p>\n<p><strong>Action</strong> <span class=\"pf2-icon\">2</span> command</p>\n<hr />\n<p><strong>Effect</strong> The <em>wayfinder</em>, glows with a spiritual radiance, revealing that which is @UUID[Compendium.pf2e.conditionitems.Hidden]{Hidden}. This has the effects of @UUID[Compendium.pf2e.spells-srd.Faerie Fire]{Faerie Fire}, except the effect occurs in a @Template[type:emanation|distance:10]{10-foot radius} centered around you.</p>\n<p>You can safely use this benefit once per adventure. You can attempt to use it a second time, overcharging the <em>wayfinder</em> at the risk of destroying it. When you do so, roll a @Check[type:flat|dc:10]. On a success, the <em>wayfinder</em> is broken. On a failure, the <em>wayfinder</em> is destroyed. If anyone tries to overcharge a <em>wayfinder</em> that's already been overcharged, the item is automatically destroyed (even if it has been repaired) and does not provide the activated benefit.</span></p>\n<hr />\n<p><strong>Special:</strong> You can apply multiple upgrades to a single <em>wayfinder</em>. Each upgrade only affects one <em>wayfinder</em> in your possession, chosen when the boon is purchased. If a <em>wayfinder</em> has multiple upgrades on it, you can only safely use each ability once per scenario.</p>\n<p>If the <em>wayfinder</em> with this upgrade is destroyed, you can apply the upgrade to another <em>wayfinder</em> during Downtime for no additional cost beyond that of the <em>wayfinder</em> itself.</p>"
```

Comment:
"The <em>wayfinder</em>, glows"
Consider removing comma.

"<strong>Special:</strong>"
Consider removing colon.